aliases/ansible.yaml

```diff
@@ -3,6 +3,17 @@
 ---
 name: "chatops.ansible_local"
 action_ref: "ansible.command_local"
-description: "Run ansible command on local machine"
+description: "Run Ansible command on local machine"
 formats:
-  - "ansible {{args}}"
+  - display: "ansible <command>"
+    representation:
+      - "ansible {{ args }}"
+result:
+  format: |
+    Ansible command `{{ execution.parameters.args }}` result: {~}
+    {% if execution.result.stderr %}*Stdout:* {% endif %}
+    ```{{ execution.result.stdout }}```
+    {% if execution.result.stderr %}*Stderr:* ```{{ execution.result.stderr }}```{% endif %}
```

Comment:
@emedvedev Here, with the result message I want to include WebUI execution URL (same as URL in `ack`).
Is it possible to do?